deployment/ansible/roles/driver.app/tasks/main.yml

```diff
@@ -17,20 +17,20 @@
   command: >
     docker build
       -f {{ root_app_dir }}/Dockerfile.base
-      -t quay.io/azavea/driver-app:{{ docker_image_tag }}
+      -t driver-app:{{ docker_image_tag }}
```

Comment:
This is inconsistent with the other `docker build ...` commands in other parts of the Ansible provisioning process. Initially I made comments that those other commands should be made to match this one, but I think that substituting a blank variable for `{{docker_repository}}` like you suggested yesterday is a better approach. I think you could update this to keep the `{{docker_repository}}` prefix that the other `docker build ...` commands use and then set the blank variable in the development group_vars (see the comment above), and it should work after that.